compiler/quilt/tools/package.py

```diff
@@ -243,22 +236,21 @@ def save_df(self, dataframe, name, path, ext, target):
                 objhash = digest_file(path)
                 move(path, self._store.object_path(objhash))
                 hashes.append(objhash)
-            self._add_to_contents(buildfile, hashes, ext, path, target)
+            self._add_to_contents(node_path, hashes, ext, source_path, target)
```

Comment:
This fixes a bug: `path` gets overwritten with something unrelated in the `for` loop above.